lang/LangSource/PyrObject.cpp

```diff
@@ -110,7 +110,7 @@ PyrSymbol *s_identitydictionary;
 PyrSymbol *s_linkedlist;
 PyrSymbol *s_sortedlist;
 PyrSymbol *s_array;
-PyrSymbol *s_list, *s_method, *s_fundef, *s_frame, *s_class;
+PyrSymbol *s_list, *s_rest, *s_method, *s_fundef, *s_frame, *s_class;
```

Comment:
I'll leave it to you to revise the primitives to get hold of the Rest classobj without predefining the symbol. I suppose you'll have to `getsym("Rest")` on every invocation, which I suppose may also have a slight performance impact.

That is... I was using `s_rest` to avoid the need to do a symbol lookup for `'Rest'` for every call into the `delta` and `isRest` methods... these methods will be called at least once for every Event played by an EventStreamPlayer, so we can predict they will be called frequently, so it wouldn't hurt to do the symbol lookup once at language startup and reuse.